src/main/java/qna/domain/DeleteHistory.java

```diff
@@ -21,58 +23,24 @@ public class DeleteHistory {
     @Enumerated(value = EnumType.STRING)
     private ContentType contentType;
     private Long contentId;
-    private Long deletedById;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User deleter;
 
     @CreatedDate
     @Column(updatable = false)
     private LocalDateTime createDate;
 
-    public DeleteHistory() {
-    }
-
-    public DeleteHistory(ContentType contentType, Long contentId, Long deletedById) {
-        this.contentType = contentType;
-        this.contentId = contentId;
-        this.deletedById = deletedById;
+    protected DeleteHistory() {
     }
 
-    public DeleteHistory(ContentType contentType, Long contentId, Long deletedById, LocalDateTime createDate) {
+    public DeleteHistory(ContentType contentType, Long contentId, User deleter, LocalDateTime createDate) {
         this.contentType = contentType;
         this.contentId = contentId;
-        this.deletedById = deletedById;
+        this.deleter = deleter;
         this.createDate = createDate;
```

Comment:
```suggestion
        this.createDate = LocalDateTime.now();
```

외부에서 시간을 설정하게 만들면 생성 시간을 조작할 수 있겠네요.